index.bikeshed.bs

```diff
@@ -36,22 +36,47 @@ spec: ABNF; urlPrefix: https://tools.ietf.org/html/rfc5234
     text: VCHAR; url: appendix-B.1
     text: WSP; url: appendix-B.1
 
+spec: CSP; urlPrefix: https://w3c.github.io/webappsec-csp/
+  type: dfn
+    text: Content Security Policy; urlPrefix: #
+    text: policy; url: policy
+    text: directive; url: directives
+    text: directive value; url: directive-value
```

Comment:
I'd mark this up as `text: value; for: directive; url: directive-value`. It's not defining "directive value", it's defining "value" in the context of "directive".